### PR TITLE
Update foreman_remote_execution to 2.0.0

### PR DESCRIPTION
**Pull request by adamruzicka**

### CODE REVIEW

File: plugins/ruby-foreman-remote-execution/debian/gem.list [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # list of gem urls to download via Jenkins, space separated
-GEMS="https://rubygems.org/downloads/foreman_remote_execution-1.8.4.gem"
+GEMS="https://rubygems.org/downloads/foreman_remote_execution-2.0.0.gem"
 GEMS="$GEMS https://rubygems.org/downloads/foreman_remote_execution_core-1.2.0.gem"
```
Comment (mmoll) [inline plugins/ruby-foreman-remote-execution/debian/gem.list]: this should be 1.3.0, I guess? :)

Comment (adamruzicka) [reply plugins/ruby-foreman-remote-execution/debian/gem.list]: That's being done in here https://github.com/theforeman/foreman-packaging/pull/4166

Comment (mmoll) [reply plugins/ruby-foreman-remote-execution/debian/gem.list]: I see... :eyes:

**Comment by mmoll** [comment]
merged, díky @adamruzicka!

**Pull request merged**